scripts/sam_uncollapse.pl

```diff
@@ -92,7 +92,7 @@ sub usage {
 Comments:
 CAUTION: Only marginal validation of the input file type/format performed!
 
-Version 1.2 (2014-08-26)
+Version 1.2.1 (2023-12-08)
```

Comment:
This would at least constitute a minor version change, if not a major version change (see next comment on breaking changes).